salt/modules/hadoop.py

```diff
@@ -88,6 +88,22 @@ def dfs(command=None, *args):
         return 'Error: command must be provided'
 
 
+def dfsadmin(command=None, *args):
+    '''
+    Execute a command on DFS
+
+    CLI Example:
```

Comment:
This new function needs a `.. versionadded:: Fluorine` tag, and it would be good to add documentation for the `command` arg.